htdocs/compta/facture/card.php

```diff
@@ -3919,7 +3919,7 @@ function js_recalculate_revenuestamp(){
 	print '</tr>';
 
 	// Payments already done (from payment on this invoice)
-	$sql = 'SELECT p.datep as dp, p.ref, p.num_paiement, p.rowid, p.fk_bank,';
+	$sql = 'SELECT DISTINCT p.datep as dp, p.ref, p.num_paiement, p.rowid, p.fk_bank,';
```

Comment:
This fix looks wrong to me.
Why do you get the duplicate line here ?
Do you know which left join has effect to create duplicate ?

Reply:
@eldy 
étrangement un paiement sur une facture apparait autant de fois que l'id de l'entité sur laquelle on est !

sur l'entité 2 un paiement apparait 2 fois
sur l'entité 3 un paiement apparait 3 fois
etc...

Reply:
@eldy 
en fait c'est ce left join qui pose problème

LEFT JOIN llx_c_paiement as c ON p.fk_paiement = c.id